starter-code/contact-app/src/app/app.component.css

```diff
@@ -0,0 +1,4 @@
+.img {
```

Comment:
Maybe you prefer to put this in 'contact-list.component.css', then the rules are applied only to the images in that component.